dotCMS/src/main/java/com/dotmarketing/portlets/containers/action/EditContainerAction.java

```diff
@@ -618,16 +620,10 @@ private void saveContainerStructures (ActionRequest req, Container currentContai
 		}
 
 		//Delete MultiTree for old / unused ContainerStructures
+		final Map<String, Object> newContainerStructures = csList.stream().collect(Collectors.toMap(cs -> cs.getStructureId(), cs -> cs));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Specify a type for: 'cs' [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2211)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Specify a type for: 'cs' [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2211)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this lambda with a method reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1612)